apps/web/src/routes/AllView/AllView.jsx

```diff
@@ -102,13 +102,13 @@ export default function AllViews () {
     return widgetsSorted.map(widget => {
       const title = widget.title
       const url = widgetUrl({ widget, rootPath, groupSlug: routeParams.groupSlug, context: 'group' })
```

Comment:
widget.url({ rootPath, groupSlug: routeParams.groupSlug, context: 'group' }) or if it is predictable from widget attributes and a route object then simply widget.url(route) ? ...